### ISSUE TITLE
Add threat sheets

### ISSUE THREAD

**Issue by OfficerHalf** [OfficerHalf/cryptomancer-fvtt#62]
Should probably be done after #49

**Comment by nathonius** [comment]
Is this actually necessary? Threats are almost identical to characters in the way they function, maybe i just need an alternate sheet for the same underlying data model.

**Comment by nathonius** [comment]
Yes, it's necessary - need to be able to distinguish between players and non-players for combat at the very least.
